tests/models.py

```diff
@@ -37,6 +37,22 @@ class Meta:
 TestModel = type('TestModel', (models.Model,), model_attrs)
 
 
+class MetaWithSchema:
+    db_table = '"appschema"."testmodel"'
+    unique_together = ['id', 'name']
+
+
+model_attrs_with_schema = {
+    'name': models.CharField(max_length=50, null=True, blank=True, default=''),
+    'int_field': models.IntegerField(null=True, blank=True),
+    'objects': BulkUpdateManager(),
+    'Meta': MetaWithSchema,
+    '__module__': __name__
+}
+
+TestModelWithSchema = type('TestModelWithSchema', (models.Model,), model_attrs_with_schema)
```

Comment:
Why is model created in such an ugly way? `TestModel` is created as soon as `json_field`, `hstore_field` and `array_field` are not available in early django versions and should be created dynamically. I don't see any need for it here.
```suggestion
class TestModelWithSchema(models.Model):
    class Meta:
        db_table = '"appschema"."testmodel"'
        unique_together = ['id', 'name']
        
    name = models.CharField(max_length=50, null=True, blank=True, default='')
    int_field = models.IntegerField(null=True, blank=True)
    objects = BulkUpdateManager()
```